toke.c

```diff
@@ -775,6 +777,42 @@ S_cr_textfilter(pTHX_ int idx, SV *sv, int maxlen)
 }
 #endif
 
+STATIC void
+S_yyerror_non_ascii_message(pTHX_ const U8 * const s)
+{
+    PERL_ARGS_ASSERT_YYERROR_NON_ASCII_MESSAGE;
+
+    yyerror_pv(Perl_form(aTHX_ "Use of non-ASCII character 0x%02X"
+                               " illegal when 'use source::encoding"
+                               " \"ASCII\"' is in effect", *s), 0);
```

Comment:
I'd suggest putting a linebreak in after the `aTHX_` so you can start the string literal much closer to the left, and hence use two longer lines rather than these three tightly-wrapped ones.

Reply:
ok